unstract/metrics/README.md

```diff
@@ -0,0 +1,22 @@
+# Unstract Metrics Aggregator
+
+Helps collect metrics from Unstract and its adapters and pushes them to Elasticsearch.
+
+Run the below services with the compose profile `unstract-metrics`.
+- elasticsearch
+- cerebro (UI for managing es instance)
+
+```shell
+VERSION=<version> docker compose -f docker-compose.yaml --profile unstract-metrics up -d 
+```
+
+
+## Using Cerebro: An Elasticsearch Web Admin tool
```

Comment:
@chandrasekharan-zipstack Create a confluence page for Cerebro no need to include here.

Reply:
@jaseemjaskp Wondering if we should mention it in README so that users can install for admin tasks if they wish? Are there any other recommended tools?

Reply:
@hari-kuriakose kibana maybe?

Reply:
@hari-kuriakose @chandrasekharan-zipstack Elasicsearch official one is Kibana. we can  mention that in readme if required. https://github.com/lmenezes/cerebro  is not up to date with official ES.

Reply:
@jaseemjaskp if cerebro is not upto date with official ES do you think we should also try kibana?

Reply:
@chandrasekharan-zipstack try out Kibana. but no need into include in docker-compose file